src/group/libp2p_group_gossip_server.erl

```diff
@@ -479,7 +511,20 @@ maybe_lookup_seed_in_dns(TargetAddrs) ->
         false ->
             TargetAddrs;
         true ->
-            lookup_seed_from_dns(TargetAddrs)
+            case get(dns_seeds) of
+                undefined ->
+                    Res = lookup_seed_from_dns(TargetAddrs),
+                    put(dns_seeds, {erlang:system_time(seconds), Res}),
+                    Res;
+                {Time, Value} ->
+                    case (erlang:system_time(seconds) - Time) > 60 of
```

Comment:
This cache time should be a macro at the least, if not configurable from an env var